install/install.sh

```diff
@@ -9,11 +9,47 @@ REMOTE="$REMOTE/$VERSION"
 set -e
 
 # Check if the script is running in ARM architecture
-[[ "$(uname -m)" != "arm"* ]] && echo "Companion only supports ARM computers."
+[[ "$(uname -m)" != "arm"* ]] && (
+    echo "Companion only supports ARM computers."
+    exit 1
+)
 
 # Check if the script is running as root
 [[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
 
+# Running in a BCM27* or BCM28, do necessary Raspberry configuration
+cat /proc/cpuinfo | grep -Eq "BCM(27|28)" && (
+    echo "Configuring board.."
```

Comment:
```suggestion
    echo "Raspberry Pi detected, configuring board.."
```

Reply:
This should work with any BCM27 and 28 SBC, not necessary raspberry.

Reply:
anyway, I think we should tell the user which configuration is being applied

Reply:
well, maybe in the next pr.

Reply:
> anyway, I think we should tell the user which configuration is being applied

Check the following lines.